components/design-system/package.json

```diff
@@ -33,14 +33,16 @@
   },
   "dependencies": {
     "@nice-digital/icons": "^2.2.7",
+    "@nice-digital/nds-a-z-list": "^0.2.0-alpha.0",
     "@nice-digital/nds-action-banner": "^1.2.14",
     "@nice-digital/nds-alert": "^1.2.12",
+    "@nice-digital/nds-alphabet": "^0.3.0-alpha.0",
     "@nice-digital/nds-breadcrumbs": "^1.2.14",
     "@nice-digital/nds-button": "^1.2.12",
     "@nice-digital/nds-callout": "^1.1.14",
     "@nice-digital/nds-card": "^1.2.15",
-    "@nice-digital/nds-checkbox": "^1.2.12",
-    "@nice-digital/nds-column-list": "^1.1.0",
+    "@nice-digital/nds-checkbox": "^1.3.0-alpha.0",
```

Comment:
That's weird checkbox has changed!

Reply:
Yep - I spotted a typo in the docs and corrected it... ultra pedantic